calc/calc.py

```diff
@@ -52,6 +58,72 @@ def __init__(self, text, position=0, current_token=None):
         self.position = position
         self.current_token = current_token
 
+    def _tokenize_integer(self, text):
+        """If have found a single digit then this helper function will read
+        characters until the end of the integer is found.
+
+        args:
```

Comment:
`args` is google standard but I'll take project consistency over an outside standard. I'm also a little partial to capital but my team prefers small.